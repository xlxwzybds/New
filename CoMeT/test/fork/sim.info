{'cmdline': ['../../run-sniper',
             '-n',
             '2',
             '-c',
             'gainestown',
             '--sift',
             '--',
             './fork'],
 'git_revision': '4ef841cd19f7a43fe2b912893f039737fe24b11a',
 'host': 'shailja-OptiPlex-7050',
 'pin_version': 'unknown',
 'rusage': (0.21967099999999998,
            0.034457999999999996,
            36336,
            0,
            0,
            0,
            7505,
            0,
            0,
            80,
            168,
            0,
            0,
            0,
            5472,
            1584),
 'snipercmd': ['bash',
               '-c',
               '/home/shailja/temp/sniper_7.3_major_bug_fixing/lib/sniper -c /home/shailja/temp/sniper_7.3_major_bug_fixing/config/base.cfg --general/total_cores=2 --general/output_dir=/home/shailja/temp/sniper_7.3_major_bug_fixing/test/fork --config=/home/shailja/temp/sniper_7.3_major_bug_fixing/config/nehalem.cfg --config=/home/shailja/temp/sniper_7.3_major_bug_fixing/config/gainestown.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpvMMLyV/run_benchmarks'],
 't_elapsed': 3.5921409130096436,
 't_start': 1606311829.856935,
 'tracecmds': [['/home/shailja/temp/sniper_7.3_major_bug_fixing/record-trace',
                '-o',
                '/tmp/tmpvMMLyV/run_benchmarks',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './fork']],
 'user': 'shailja',
 'vmem': 42524672L}
