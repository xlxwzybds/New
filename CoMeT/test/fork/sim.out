                                     | Core 0     | Core 1    
  Instructions                       |      45269 |       2531
  Cycles                             |     136990 |     136990
  IPC                                |       0.33 |       0.02
  Time (ns)                          |      51500 |      51500
  Idle time (ns)                     |       2019 |      44515
  Idle time (%)                      |       3.9% |      86.4%
Branch predictor stats               |            |           
  num correct                        |      10067 |        358
  num incorrect                      |       1706 |        120
  misprediction rate                 |     14.49% |     25.10%
  mpki                               |      37.69 |      47.41
TLB Summary                          |            |           
  I-TLB                              |            |           
    num accesses                     |       5139 |        377
    num misses                       |         49 |         24
    miss rate                        |      0.95% |      6.37%
    mpki                             |       1.08 |       9.48
  D-TLB                              |            |           
    num accesses                     |      11133 |       1025
    num misses                       |         24 |         16
    miss rate                        |      0.22% |      1.56%
    mpki                             |       0.53 |       6.32
  L2 TLB                             |            |           
    num accesses                     |         73 |         40
    num misses                       |         73 |         40
    miss rate                        |    100.00% |    100.00%
    mpki                             |       1.61 |      15.80
Cache Summary                        |            |           
  Cache L1-I                         |            |           
    num cache accesses               |       5139 |        377
    num cache misses                 |        573 |        178
    miss rate                        |     11.15% |     47.21%
    mpki                             |      12.66 |      70.33
  Cache L1-D                         |            |           
    num cache accesses               |      11133 |       1025
    num cache misses                 |        346 |         77
    miss rate                        |      3.11% |      7.51%
    mpki                             |       7.64 |      30.42
  Cache L2                           |            |           
    num cache accesses               |        950 |        273
    num cache misses                 |        916 |        254
    miss rate                        |     96.42% |     93.04%
    mpki                             |      20.23 |     100.36
  Cache L3                           |            |           
    num cache accesses               |        947 |        272
    num cache misses                 |        916 |        254
    miss rate                        |     96.73% |     93.38%
    mpki                             |      20.23 |     100.36
DRAM summary                         |            |           
  num dram accesses                  |       1170 |          0
  average dram access latency (ns)   |      96.33 |        inf
  average dram queueing delay        |      42.90 |       0.00
  average dram bandwidth utilization |     19.00% |      0.00%
Coherency Traffic                    |            |           
  num loads from dram                |        191 |         52
  num loads from dram cache          |          0 |          0
  num loads from remote cache        |          0 |          0
